datumaro/plugins/data_formats/voc/exporter.py

```diff
@@ -592,9 +605,11 @@ def save_label_map(self):
             write_label_map(path, self._label_map)
 
     def _load_categories(self, label_map_source):
-        if label_map_source == LabelmapType.voc.name:
-            # use the default VOC colormap
-            label_map = make_voc_label_map()
+        if (
+            label_map_source in [t.name for t in LabelmapType]
+            and label_map_source != LabelmapType.source.name
+        ):
+            label_map = make_voc_label_map(task=list(self._task)[0])
```

Comment:
The naming change (`self._tasks` -> `self._task`) seems that it restricts `self._task` to be singular one. However, L141-145, using set operation (`self._task & {VocTask.action_classification}`), and this line `list(self._task)[0]` seems confusing because it maintains `set` data type (plural).

Reply:
The task has been updated to be single.